src/partielspy/plugin_list.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+from .plugin import Plugin
+from .plugin_key import PluginKey
+
+
+class PluginList(list[Plugin]):
+    @classmethod
+    def from_json(cls, data: json):
+        plugin_list = cls()
+        for plugin in data:
+            key = plugin.get("key", "")
```

Comment:
It would be more consistent to add a from_json method to PluginKey and call this method.